pandas/core/indexes/datetimes.py

```diff
@@ -1747,7 +1747,33 @@ def freq(self, value):
     is_year_end = _field_accessor(
         'is_year_end',
         'is_year_end',
-        "Logical indicating if last day of year (defined by frequency)")
+        """
+        Return a boolean indicating whether the date is the last day of the
+        year.
+
+        Returns
+        -------
+        is_year_end : Series of boolean.
+
+        See Also
+        --------
+        is_year_start : Return a boolean indicating whether the date is the
```

Comment:
See comment https://github.com/pandas-dev/pandas/pull/20275#discussion_r173651295 (once we figure out how to handle these)